acceptance/config/common/options.rb

```diff
@@ -0,0 +1,16 @@
+{
+  :ssh                         => {
+    :keys => ["id_rsa_acceptance", "#{ENV['HOME']}/.ssh/id_rsa-acceptance"],
+  },
+  :load_path                   => './lib',
+  :xml                         => true,
+  :timesync                    => false,
+  :repo_proxy                  => true,
+  :add_el_extras               => true,
+  :forge_host                  => 'forge-aio01-petest.puppetlabs.com',
```

Comment:
good question.  it's what was always in the Rakefile, i just moved it.  :older_man: